lib/tags/feed-item.tag

```diff
@@ -33,7 +33,7 @@
                           this.on('update', function(){ this.root.innerHTML = i && i;});
                         </raw>
                     </div>
-                    <div if={i.checkMarkDownPattern(activity)}>
+                    <div if={i.checkMarkDownPattern(activity)} class="graphjs-youtube-embed">
```

Comment:
i see this, but not always there is a youtube, will this affect other cases as well?